src/common/logging.hpp

```diff
@@ -40,26 +40,27 @@
 
 #include "common/types.hpp"
 
+#ifndef OTBR_LOG_TAG
+#define OTBR_LOG_TAG ""
```

Comment:
Not sure if this is better.
```suggestion
#error "OTBR_LOG_TAG not defined!"
```

Reply:
I would prefer throwing compiling error if there is no log tag.

Reply:
Looks like we can simply remove this block.